docling/models/tesseract_ocr_cli_model.py

```diff
@@ -237,31 +249,30 @@ def __call__(
                             text = row["text"]
                             conf = row["conf"]
 
-                            l = float(row["left"])
-                            b = float(row["top"])
-                            w = float(row["width"])
-                            h = float(row["height"])
-
-                            t = b + h
-                            r = l + w
+                            rotated_bbox = (
+                                row["left"],
+                                row["top"],
+                                row["width"],
+                                row["height"],
+                            )
+                            rotated_bbox = cast(
+                                Box, tuple(float(c) for c in rotated_bbox)
+                            )
+                            bbox = tesseract_box_to_bounding_box(
+                                rotated_bbox,
+                                offset=ocr_rect,
+                                scale=self.scale,
+                                orientation=doc_orientation,
+                                rotated_image_size=high_res_image.size,
+                            )
 
                             cell = TextCell(
```

Comment:
Since the `cell.rect` can now support arbitrary rectangle shapes (at an angle), it would be very useful to populate this rect with the correct orientation (instead of using `BoundingRectangle.from_bounding_box(...)`, which ignores any orientation.